ncm-postfix/src/main/pan/components/postfix/schema.pan

```diff
@@ -462,6 +462,7 @@ type postfix_main = {
     "smtp_tls_note_starttls_offer" ? boolean
     "smtp_tls_per_site" ? string
     "smtp_tls_policy_maps" ? string
+    "smtp_tls_protocols" ? string
```

Comment:
It would be nice to validate this, although I suspect the list would require regular maintenance as valid protocols are added and removed?